src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("the truck started it`s work");
```

Comment:
The message in the `System.out.println` should be more informative and indicate the type of machine. For example, "Truck started its work".